src/middleware/log.ml

```diff
@@ -339,90 +347,114 @@ let initialize_log
 
   enable := enable_;
   level := level_;
+  let `Initialized = initialized () in
+  ()
+
+module Make (Pclock : Mirage_clock.PCLOCK) = struct
```

Comment:
Please separate this into two commits, one commit that does not indent a large chunk of code, and a second commit "Adjust whitespace after preceding commit"

i.e. wrap in the `Make` functor without indenting the parts of the body that consist of pre-existing code.

GitHub is not doing a good job of showing whitespace-only changes right now for some reason, and it makes it challenging to review this PR reading a large amount of non-changes presented as changes line-by-line.

It's also the right thing to do for future usage of blame, which I do use a lot as a form of documentation.

Reply:
Even better would be not to include the "Adjust whitespace" commit, since we might tweak the PR multiple times. I'll just add  the adjustment at the end, after merging the PR.

Reply:
I did something with a better diff, can I force push the first commit?

Reply:
Or I can just push a new commit like: "Re-adjust whitespace and newline"

Reply:
> Or I can just push a new commit like: "Re-adjust whitespace and newline"

This is better. If you force push the first commit, you effectively undo all the incremental work I've done, since I don't know anymore what changed. If you just add a new commit, I can choose to inspect that commit, and then inspect the net change from the start of the PR for just the files I haven't reviewed yet due to the diff (and GitHub apparently not displaying whitespace changes like it used to, at least in its dark theme).

Reply:
Done ✅

Reply:
Thanks, looks much more reviewable now!